identity/tests/db_queries.rs

```diff
@@ -149,6 +149,34 @@ async fn db_get_user_not_exists() {
     assert_eq!(Err(Error::NotFound), result);
 }
 
+// get an user by its sub
+#[tokio::test]
+async fn db_get_user_by_sub() {
+    // Arrange
+    let (_configuration, db_pool, _db_test_context) = setup(stdext::function_name!().into())
```

Comment:
```suggestion
    let (_configuration, db_pool, _db_test_context) = setup("db_get_user_by_sub")
```

Reply:
Will stick for now to the macro, but will look further into it.